### PR TITLE
Migrated from TD

### PR DESCRIPTION
**Pull request by ilfrin**
not mergeable, preview

### CODE REVIEW

File: prestodb/hdp2.5-base/Dockerfile [hunk 1]
```diff
@@ -35,7 +35,10 @@ RUN \
     hive-metastore \
     hive-server2 \
 
-  && yum install -y python-setuptools && easy_install supervisor \
+  && yum install -y python-setuptools \ 
+    && yum install -y epel-release \
+    && yum install -y python-pip && pip install pip --upgrade \
```
Comment (kokosing) [inline prestodb/hdp2.5-base/Dockerfile]: why do you upgrade `pip`?

File: prestodb/hdp2.5-base/Dockerfile [hunk 2]
```diff
@@ -35,7 +35,10 @@ RUN \
...
+    && pip install --no-deps --ignore-installed --pre supervisor
```
Comment (kokosing) [inline prestodb/hdp2.5-base/Dockerfile]: why `--no-deps`, `--ignore--installed` and `--pre`

File: prestodb/cdh5-hive/files/root/setup.sh [hunk 3]
```diff
@@ -45,14 +48,5 @@ sleep 10s
...
-cp /tmp/hadoop_conf/core-site.xml /etc/hadoop/conf
-cp /tmp/hadoop_conf/mapred-site.xml /etc/hadoop/conf
-cp /tmp/hadoop_conf/yarn-site.xml /etc/hadoop/conf
-cp /tmp/hadoop_conf/hadoop-env.sh /etc/hadoop/conf
-cp /tmp/hadoop_conf/hive-env.sh /etc/hive/conf
-rm -r /tmp/hadoop_conf
-
-# 9 Init zookeeper
+# 8 Init zookeeper
 /etc/init.d/zookeeper-server init
```
Comment (kokosing) [inline prestodb/cdh5-hive/files/root/setup.sh]: do we need zookeeper?

**Comment by ilfrin** [comment]
rebased and moved the supervisor installation to the base OS image

File: prestodb/hdp2.5-base/Dockerfile [hunk 4]
```diff
@@ -33,10 +33,6 @@ RUN \
 
     hive \
     hive-metastore \
-    hive-server2 \
-
-  && yum install -y epel-release && yum install -y python-pip && pip install supervisor \
-
-  && yum -y clean all && rm -rf /tmp/* /var/tmp/*
```
Comment (kokosing) [inline prestodb/hdp2.5-base/Dockerfile]: do not remove cleanup

File: prestodb/centos6-oj8/Dockerfile [hunk 5]
```diff
@@ -45,6 +45,10 @@ RUN \
            $JDK_PATH/jre/lib/amd64/libgstreamer-lite.so \
            $JDK_PATH/jre/lib/amd64/libjavafx*.so \
            $JDK_PATH/jre/lib/amd64/libjfx*.so && \
+    
+    # install supervisor
+    yum install -y epel-release && yum install -y python-pip && pip install supervisor && \
```
Comment (kokosing) [inline prestodb/centos6-oj8/Dockerfile]: you need to remove it also from centos6-oj8-ldap

File: prestodb/cdh5-base/Dockerfile [hunk 6]
```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM teradatalabs/centos6-java8-oracle:unlabelled
-MAINTAINER Teradata Docker Team <docker@teradata.com>
+FROM prestodb/centos6-oj8:unlabelled
+MAINTAINER Presto community <https://prestodb.io/community.html>
```
Comment (kokosing) [inline prestodb/cdh5-base/Dockerfile]: the best would be to do a single change per commit. Here is plenty of changes... which makes it hard to review.

File: prestodb/cdh5-base/Dockerfile [hunk 7]
```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM teradatalabs/centos6-java8-oracle:unlabelled
-MAINTAINER Teradata Docker Team <docker@teradata.com>
+FROM prestodb/centos6-oj8:unlabelled
```
Comment (kokosing) [inline prestodb/cdh5-base/Dockerfile]: Don't you need to add `flag/cdh-base.flags` file here and for other images as well?

File: prestodb/centos6-oj8-openldap/Dockerfile [hunk 8]
```diff
@@ -73,6 +73,7 @@ RUN keytool -import -alias caroot -storepass testldap -keystore /etc/openldap/ce
     -file /etc/openldap/certs/caroot.cer
 
 # Install supervisor.d
-RUN yum install -y epel-release && yum install -y python-pip && pip install supervisor && yum -y clean all && rm -rf /tmp/* /var/tmp/*
+RUN yum install -y setuptools epel-release && yum install -y python-pip && pip install --upgrade pip && pip install --upgrade setuptools && pip install supervisor && \
```
Comment (kokosing) [inline prestodb/centos6-oj8-openldap/Dockerfile]: [nit] make && to end the line. That way it will be easier to read.

File: prestodb/cdh5-hive/files/etc/hadoop/conf/capacity-scheduler.xml [hunk 9]
```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
```
Comment (kokosing) [inline prestodb/cdh5-hive/files/etc/hadoop/conf/capacity-scheduler.xml]: it looks like a fixup commit

File: prestodb/cdh5-hive/Dockerfile [hunk 10]
```diff
@@ -59,4 +59,4 @@ EXPOSE 9083 10000
 # SOCKS PORT
 EXPOSE 1180
 
-CMD supervisord -c /etc/supervisord.conf
+CMD echo "127.0.0.1 hadoop-master" >> /etc/hosts && supervisord -c /etc/supervisord.conf
```
Comment (kokosing) [inline prestodb/cdh5-hive/Dockerfile]: Please undo this change.

Docker images are run from docker-compose, please see `presto-product-tests/conf/docker/singlenode/compose.sh config` as an example

To change the docker image for haddop there you need to:
```
 export HADOOP_MASTER_IMAGE='teradatalabs/cdh5-hive:23'
```

File: prestodb/cdh5-base/Dockerfile (hunk 7 above)
Comment (kokosing) [reply prestodb/cdh5-base/Dockerfile]: Nevermind this commit. These files are created dynamically.

File: prestodb/cdh5-hive/files/etc/hadoop/conf/capacity-scheduler.xml (hunk 9 above)
Comment (ilfrin) [reply prestodb/cdh5-hive/files/etc/hadoop/conf/capacity-scheduler.xml]: yup will squash it with the CDH commit

File: prestodb/cdh5-base/Dockerfile (hunk 6 above)
Comment (ilfrin) [reply prestodb/cdh5-base/Dockerfile]: sorry, will behave better next time

File: prestodb/cdh5-hive/Dockerfile (hunk 10 above)
Comment (ilfrin) [reply prestodb/cdh5-hive/Dockerfile]: discussed this offline, OK
this was needed in order to run this image with plain `docker run` command, dropping the change

File: prestodb/centos6-oj8-openldap/Dockerfile (hunk 8 above)
Comment (ilfrin) [reply prestodb/centos6-oj8-openldap/Dockerfile]: that was following your last changes, but sure, will change this :)

File: prestodb/cdh5-hive/files/root/setup.sh (hunk 3 above)
Comment (ilfrin) [reply prestodb/cdh5-hive/files/root/setup.sh]: no idea

**Comment by ilfrin** [comment]
applied comments

**Comment by kokosing** [comment]
Lets ship it, we will fix bugs in following commits
